ingredient/views.py

```diff
@@ -1,3 +1,47 @@
-from django.shortcuts import render
+import json
+from user.models import User
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from ingredient.models import Ingredient
+
+class IngredientcategoriesView(View):
```

Comment:
`IngredientsView`와 반환하는 값이 같은 것 같은데, 왜 두개로 나누어져 있나요?

```suggestion
class IngredientcategoriesView(View):
```